### DISCUSSION TITLE
Edge Panel for Samsung phones

### DISCUSSION THREAD

**Discussion by DanStevens** [stratumauth/app#919] [Ideas]
The Samsung One UI has a feature called ['Edge Panels'](https://www.samsung.com/uk/support/mobile-devices/what-is-the-edge-panel-and-how-do-i-use-it/), which hosts special 'thin' apps in panel that can be accessed by sliding right from the edge of the phone screen, without needing to switch away from the current app. This [YouTube video](https://www.youtube.com/shorts/kWsVA1K0h98) explains it well.

For long time I thought that an edge panel would be a good host for a OTP authenticator app for when you need a OTP for an app on the phone itself. It would be so nice to quickly select and copy a OTP to the clipboard without needing to switch away from the current app, 

I've been looking into how to develop edge panels using the [Samsung Edge SDK](https://developer.samsung.com/galaxy-edge/overview.html) so that I could create Quick View or Quick Control panel app that could host a list of OTPs myself. I've learnt that developing an edge panel is very similar to developing a widget, and I'm at a point where I could create generic edge panel app.

The next step for me is learning how to implement the OTP authenticator functionality, which is where I came across this project. So I thought I'd let you all know I'm looking into this, and see what interest there might be in this as a concept, as I'm likely to draw on the code of this project in some way.

Authenticator Pro is a nice app so if I'm able to turn my idea into a working proof-of-concept, I'd be very open to contributing to this project if this would be interest to the owners or community.

**Comment by BroPro40** [comment]
Howdy, did you success in your development?

**Comment by spiralgang** [comment]
Kinda me too sorta how'd you do ?
